src/ResponseCache.php

```diff
@@ -90,4 +90,14 @@ protected function addCachedHeader(Response $response): Response
 
         return $clonedResponse;
     }
+
+    public function forget(string $uri)
```

Comment:
Could you make this handle both `strings` and `arrays` so users can forget multiple URIs in one go.

Reply:
Good idea ! I'll do that asap :)